src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -543,5 +544,22 @@
 							"\t\tbroadcast \"%loop-block% was absorbed by a sponge\"!")
 					.since("2.5");
 		}
+		if (Skript.classExists("org.bukkit.event.entity.EntityTransformEvent")) {
+			Skript.registerEvent("Entity Transform", SimpleEvent.class, EntityTransformEvent.class, "entity transform[ation]")
+			.description("Called before an entity transforms into other another entity/entities. This event triggers for:",
+					"- villager to zombie villager (infection)",
```

Comment:
Consider using HTML to format a proper list. This will *not* look pretty in generated docs page.

Reply:
In retrospect, maybe it's better if I limit the list to just 2-3 elements, and make it so that it's more generic and less likely to cause future documentation compatibility issues (if Mojang ever decides to add more transformations and or remove some transformations)